charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -35,6 +63,15 @@ spec:
   {{- if not (dig "data" .Values.rhdhOperator.secretRef.backstage.backendSecret "" $secret) }}
     {{ fail (printf "Backend secret key %s not found in secret '%s' for Backstage. Please ensure that the value key 'rhdhOperator.secretRef.backstage.backendSecret' matches the key in the secret: %s" .Values.rhdhOperator.secretRef.backstage.backendSecret .Values.rhdhOperator.secretRef.name ($secret|toPrettyJson) )}}
   {{- end }}
+
+{{- $GH_INTEGRATION_ENABLED := and .Values.rhdhOperator.secretRef.github.token (dig "data" .Values.rhdhOperator.secretRef.github.token "" $secret) }}
+{{- $GH_AUTH_ENABLED := and 
+      (and .Values.rhdhOperator.secretRef.github.clientId (dig "data" .Values.rhdhOperator.secretRef.github.clientId "" $secret))
+      (and .Values.rhdhOperator.secretRef.github.clientSecret (dig "data" .Values.rhdhOperator.secretRef.github.clientSecret "" $secret))
+}}
+
+{{- $OIDC_BASE_URL := default (printf "https://keycloak-%s.%s/auth" $.Values.rhsso.namespace (include "cluster.domain" .)) $.Values.rhsso.baseURL }}
+{{- $OIDC_METADATA_URL := default (printf "%s/realms/%s/.well-known/openid-configuration " $OIDC_BASE_URL .Values.rhsso.realm.name) $.Values.rhsso.baseURL }}
```

Comment:
same here. Check if the `rhsso.baseURL` has been provided and fail otherwise. Thoughts?